### ISSUE TITLE
datacarpentry/R-ecology-lesson

### ISSUE THREAD

**Issue by zkamvar** [carpentries/lesson-transition#55]
tracking issue for https://github.com/datacarpentry/R-ecology-lesson'

preview: https://fishtree-attempt.github.io/R-ecology-lesson/

 - [x] "blockquote" divs (my parser's fault)
 - [x] investigate figure in index
 - [x] fix https
 - [x] fix index link references

```markdown
615:episodes/03-dplyr.Rmd:703 [unknown div] blockquote
616:episodes/05-r-and-databases.Rmd:264 [unknown div] blockquote
617:index.Rmd:14 [unknown div] <div style="text-align: center; margin-top: 30px; margin-bottom: 30px;">
625:episodes/01-intro-to-r.Rmd:71 [uninformative link text]: [here](https://stat.ethz.ch/R-manual/R-devel/library/base/html/Reserved.html)
626:episodes/02-starting-with-data.Rmd:186 [uninformative link text]: [here](https://tibble.tidyverse.org/)
627:episodes/02-starting-with-data.Rmd:199 [image missing alt-text]: ./fig/data-frame.svg
628:episodes/02-starting-with-data.Rmd:716 [uninformative link text]: [here](https://www.tidyverse.org/)
629:episodes/03-dplyr.Rmd:561 [image missing alt-text]: fig/tidyr-pivot_wider_longer.gif
630:episodes/03-dplyr.Rmd:600 [image missing alt-text]: fig/pivot_wider_graphic.png
631:episodes/03-dplyr.Rmd:640 [image missing alt-text]: fig/pivot_longer_graphic.png
632:episodes/05-r-and-databases.Rmd:57 [needs HTTPS]: [a website](http://db.rstudio.com/)
633:CITATION.Rmd:65 [needs HTTPS]: [http://esapubs.org/archive/ecol/E090/118/](http://esapubs.org/archive/ecol/E090/118/)
634:CONTRIBUTING.Rmd:136 [uninformative link text]: [here](https://yihui.org/knitr/options)
635:index.Rmd:16 [missing file]: [](./fig/DC-logo-vision.png) [image missing alt-text]: ./fig/DC-logo-vision.png
636:index.Rmd:40 [missing file]: [Before we start](00-before-we-start.html)
637:index.Rmd:41 [missing file]: [Introduction to R](01-intro-to-r.html)
638:index.Rmd:42 [missing file]: [Starting with data](02-starting-with-data.html)
639:index.Rmd:43 [missing file]: [Manipulating, analyzing and exporting data with tidyverse](03-dplyr.html)
640:index.Rmd:44 [missing file]: [Data visualization with ggplot2](04-visualization-ggplot2.html)
641:index.Rmd:45 [missing file]: [SQL databases and R](05-r-and-databases.html)
642:index.Rmd:201 [uninformative link text]: [here](https://datacarpentry.org/R-ecology-lesson/CITATION)
643:instructors/instructor-notes.md:29 [missing file]: [code handout](files/code-handout.R)
644:instructors/instructor-notes.md:185 [needs HTTPS]: [http://colorbrewer2.org](http://colorbrewer2.org)
```
NOTE: this lesson is in a special status because of the work that MCM has done on the rewrite.

**Comment by tobyhodges** [comment]
@zkamvar please can you confirm whether we update should styles for this lesson or not? It has a very different repository structure from the others.

**Comment by zkamvar** [comment]
absolutely not. This has no relation to styles.

**Comment by zkamvar** [comment]
fixed in 7bcaacd59d79a991077a8a7f35b74270e7fdcbd9
